app/views/admin/content_pages/_form.html.erb

```diff
@@ -28,7 +28,8 @@
         <div class="fixed-footer">
           <%= form.govuk_submit "Save" %>
           <% if action_name === "edit" || action_name === "update" %>
-            <%= link_to 'Cancel', versions_content_page_path(content_page), data: { confirm: "Any changes will be lost, are you sure ?" }, class: "govuk-link" %>
+            <%= link_to 'Cancel', edit_admin_content_page_path(content_page), :data => {:confirm => "Are you sure you want to undo your changes ?", :class => "govuk-link"}%> |
+            <%= link_to 'Back', versions_admin_content_page_path(content_page), :data => {:confirm => "Any changes will be lost, are you sure ?", :class => "govuk-link"}%>
```

Comment:
Again - this is reintroducing a Back button that was removed in this PR https://github.com/DFE-Digital/early-years-foundation-reform/pull/530